WakaTime/Extensions/ProcessExtension.swift

```diff
@@ -0,0 +1,16 @@
+import Foundation
+
+extension Process {
+    // Runs process.launch() prior to macOS 13 or process.run() on macOS 13 or newer.
+    // Adds Swift exception handling to process.launch().
+    func execute() throws {
+        if #available(macOS 13.0, *) {
+            // Use Process.run() on macOS 13 or newer. Process.run() throws Swift exceptions.
+            try self.run()
```

Comment:
We need to catch and log these exceptions to `Console.app` and in the future also Crashlytics (#87).